package.json

```diff
@@ -118,7 +118,7 @@
     "eslint-plugin-unused-imports": "^4.1.4",
     "jest": "^29.5.0",
     "prettier": "^3.0.0",
-    "prisma": "^6.3.0",
+    "prisma": "^6.4.0",
```

Comment:
Only thing I'd say we could consider is doing this update after launch, just in case there's some unforeseen side-effects with it, being this close to launch and all...